blue_modules/libportal-react-native/libportal-react-native.podspec

```
@@ -0,0 +1,20 @@
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name         = "libportal-react-native"
+  s.version      = package["version"]
+  s.summary      = package["description"]
+  s.homepage     = package["homepage"]
+  s.license      = package["license"]
+  s.authors      = package["author"]
+
+  s.platforms    = { :ios => min_ios_version_supported }
+  s.source       = { :git => "https://github.com/afilini/libportal-react-native.git", :tag => "#{s.version}" }
```

rubocop

[convention] Prefer `to_s` over string interpolation.

(Style/RedundantInterpolation)